docs/docs/introduction.md

```diff
@@ -0,0 +1,47 @@
+---
+slug: /
+displayed_sidebar: GettingStartedSidebar
+title: ''
+---
+
+# Welcome to Ingonyama's Developer Documentation
+
+Ingonyama is a next-generation semiconductor company, focusing on Zero-Knowledge Proof hardware acceleration. We build accelerators for advanced cryptography, unlocking real-time applications. Our focus is on democratizing access to compute intensive cryptography and making it accessible for developers to build on top of.
+
+Currently our flagship products are:
+
+- **ICICLE**:
+  [ICICLE](https://github.com/ingonyama-zk/icicle) is a fully featured GPU accelerated cryptography library for building ZK provers. ICICLE allows you to accelerate your ZK existing protocols in a matter of hours or implement your protocol from scratch on GPU.
+
+---
+
+## Our current take on hardware acceleration
+
+We believe GPUs are as important for ZK as for AI.
+
+- GPUs are a perfect match for ZK compute - around 97% of ZK protocol runtime is parallel by nature.
+- GPUs are simple for developers to use and scale compared to other hardware platforms.
+- GPUs are extremely competitive in terms of power / performance and price (3x cheaper).
```

Comment:
Here it's really unclear 3x compared to what. If you follow the link two lines lower it becomes clear that the comparison is with FPGAs. I think worth writing this explicitly.